game/state/city/vequipment.cpp

```diff
@@ -89,22 +101,27 @@ bool VEquipment::fire(GameState &state, Vec3<float> targetPosition, Vec3<float>
 
 	auto fromScaled = vehicleMuzzle * VELOCITY_SCALE_CITY;
 	auto toScaled = targetPosition * VELOCITY_SCALE_CITY;
-	City::accuracyAlgorithmCity(state, fromScaled, toScaled, type->accuracy + owner->getAccuracy(),
-	                            targetVehicle && targetVehicle->isCloaked());
 
-	Vec3<float> velocity = toScaled - fromScaled;
-	velocity = glm::normalize(velocity);
-	// I believe this is the correct formula
-	velocity *= type->speed * PROJECTILE_VELOCITY_MULTIPLIER;
+	for (number_of_shots; number_of_shots > 0; number_of_shots--)
+	{
+		City::accuracyAlgorithmCity(state, fromScaled, toScaled,
+		                            type->accuracy + owner->getAccuracy(),
+		                            targetVehicle && targetVehicle->isCloaked());
+
+		Vec3<float> velocity = toScaled - fromScaled;
+		velocity = glm::normalize(velocity);
+		// I believe this is the correct formula
+		velocity *= type->speed * PROJECTILE_VELOCITY_MULTIPLIER;
 
-	auto projectile = mksp<Projectile>(
-	    type->guided ? Projectile::Type::Missile : Projectile::Type::Beam, owner, targetVehicle,
-	    homingPosition, muzzle, velocity, type->turn_rate, type->ttl, type->damage, /*delay*/ 0,
-	    /*depletion rate*/ 0, type->tail_size, type->projectile_sprites, type->impact_sfx,
-	    type->explosion_graphic, state.city_common_image_list->projectileVoxelMap, type->stunTicks,
-	    type->splitIntoTypes, manual);
-	owner->tileObject->map.addObjectToMap(projectile);
-	owner->city->projectiles.insert(projectile);
+		auto projectile = mksp<Projectile>(
+		    type->guided ? Projectile::Type::Missile : Projectile::Type::Beam, owner, targetVehicle,
```

Comment:
I know it was like that previously, but this is getting *very* busy. Possibly pulling out the ternary expression would help?

eg. add a like like:
` const auto projectile_type = type->isGuided ? Projectile::Type::Missile : Projectile::Type::Beam;`

then use the "projectile_type" variable as the first arg would help.

Everything else appears to be trivial expressions, but I could see that being missed from a quick scan.